app/javascript/widget/mixins/nextAvailabilityTime.js

```diff
@@ -0,0 +1,234 @@
+        'Wednesday',
+        'Thursday',
+        'Friday',
+        'Saturday',
+      ],
+      timeSlots: [...defaultTimeSlot],
+      timeSlot: {},
+    };
+  },
+  computed: {
+    channelConfig() {
+      return window.chatwootWebChannel;
+    },
+    workingHours() {
+      return this.channelConfig.workingHours;
+    },
+    timeZoneValue() {
+      return this.channelConfig.timezone;
+    },
+    currentDayWorkingHours() {
+      return this.workingHours.find(
+        slot => slot.day_of_week === this.currentDay()
+      );
+    },
+    nextDayWorkingHours() {
+      let nextDay = this.getNextDay(this.currentDay());
+      let nextWorkingHour = this.getNextWorkingHour(nextDay);
+
+      // It gets the next working hour for the next day. If there is no working hour for the next day,
+      // it keeps iterating through the days of the week until it finds the next working hour.
+      while (!nextWorkingHour) {
+        nextDay = this.getNextDay(nextDay);
+        nextWorkingHour = this.getNextWorkingHour(nextDay);
+      }
+      return nextWorkingHour;
+    },
+    currentDayTimings() {
+      const {
+        open_hour: openHour,
+        open_minutes: openMinute,
+        close_hour: closeHour,
+      } = this.currentDayWorkingHours ?? {};
+      return {
+        openHour,
+        openMinute,
+        closeHour,
+      };
+    },
+    nextDayTimings() {
+      const { open_hour: openHour, open_minutes: openMinute } =
+        this.nextDayWorkingHours ?? {};
+      return {
+        openHour,
+        openMinute,
+      };
+    },
+    dayDiff() {
+      // Here this is used to get the difference between current day and next working day
+      const currentDay = this.currentDay();
+      const nextDay = this.nextDayWorkingHours.day_of_week;
+      const totalDays = 6;
+      return nextDay > currentDay
+        ? nextDay - currentDay
+        : totalDays - currentDay + nextDay;
+    },
+    dayNameOfNextWorkingDay() {
+      return this.dayNames[this.nextDayWorkingHours.day_of_week];
+    },
+    hoursAndMinutesBackInOnline() {
+      if (this.presentHour() >= this.currentDayTimings.closeHour) {
+        return this.getHoursAndMinutesUntilNextDayOpen(
+          this.nextDayWorkingHours.open_all_day
+            ? 0
+            : this.nextDayTimings.openHour,
+          this.nextDayTimings.openMinute,
+          this.currentDayTimings.closeHour
+        );
+      }
+      return this.getHoursAndMinutesUntilNextDayOpen(
+        this.currentDayTimings.openHour,
+        this.currentDayTimings.openMinute,
+        this.currentDayTimings.closeHour
+      );
+    },
+    exactTimeInAmPm() {
+      return `${this.timeSlot.day !== this.currentDay() ? 'tomorrow' : ''} ${
+        this.timeSlot.day !== this.currentDay()
+          ? ''
+          : `at ${this.timeSlot.from}`
+      }`;
+    },
+    hoursAndMinutesLeft() {
+      const { hoursLeft, minutesLeft } = this.hoursAndMinutesBackInOnline;
+
+      const timeLeftChars = ['in'];
```

Comment:
This won't work when translated

Reply:
Done https://github.com/chatwoot/chatwoot/pull/6902/commits/6e3648e99934120d6af00d97e7930edf6118d984